user-creation-testing/build.gradle

```diff
@@ -1,9 +1,3 @@
-plugins {
-    id 'nva.identity.service.java-application-conventions'
-    id 'nva.identity.service.dynamodblocal'
```

Comment:
Why are you removing these from all build files? Are they redundant? Or inherited from somewhere else?

Reply:
They are inherited from somewhere else